apps/speed.c

```diff
@@ -1509,9 +1513,10 @@ int speed_main(int argc, char **argv)
 #ifndef NO_FORK
     int multi = 0;
 #endif
-#if !defined(OPENSSL_NO_RSA) || !defined(OPENSSL_NO_DSA) \
-    || !defined(OPENSSL_NO_EC)
-    long rsa_count = 1;
+#if (!defined(OPENSSL_NO_RSA) && !defined(OPENSSL_NO_DEPRECATED_3_0)) \
+    || (!defined(OPENSSL_NO_DSA) && !defined(OPENSSL_NO_DEPRECATED_3_0)) \
+    || !defined(OPENSSL_NO_EC) || !defined(OPENSSL_NO_DH)
+     long rsa_count = 1;
```

Comment:
Not your fault, but it is horrible that a variable named `rsa_count` is reused for other algorithm counts!

Reply:
Yes - so I've just renamed it to `op_count`.